tsp-output/@azure-tools/typespec-csharp/azoai/AzureOpenAI/Models/AzureCreateChatCompletionRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OpenAI.Models;
+
+namespace AzureOpenAI.Models;
+
+internal class AzureCreateChatCompletionRequest : CreateChatCompletionRequest
```

Comment:
I think this goes away entirely.

Reply:
Actually, maybe this is better than keeping a `Dictionary<string, object>` ...

Reply:
New approach is to keep these model subtypes types internal and use extension properties to retrieve values via internal properties on subtypes.